.github/workflows/renovate.yaml

```diff
@@ -0,0 +1,18 @@
+name: Run Renovate Bot
+
+on:
+  schedule:
+    - cron:  '0/30 * * * *'
+
+jobs:
+  renovate:
+    name: Renovatebot
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Self-hosted Renovate
+        uses: renovatebot/github-action@v34.29.2
+        with:
+          configurationFile: .github/renovate-config.js
+          token: ${{ secrets.RENOVATE_TOKEN }}
```

Comment:
how to create this RENOVATE_TOKEN ?

Reply:
Well its a Github access token. I've already created one that has only the "public.repo" access. I was going to suggest we create a doc which contains all of our secrets and what they do.

Reply:
yes I think that'd be good. otherwise we might lose track